### DISCUSSION TITLE
Linked record deletion [Mar 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5309] [Discord Threads]
> 🙋‍♂️ **Jakob from bunch**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Jakob from bunch**  
#### <@857998597474484295> maybe?  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### <@1006611047198761032> currently, links have to be removed first before deleting the record.    
#### let me check with the team if there is an easier alternative  
#  
> 🙋‍♂️ **charlietrigger**  
#### For anyone else searching for this: I found it quite easy to do it like so: lets say you have a table for orders and an order has many orderlines. Now you want to delete an order with id 101, but you can't because it has related orderlines. You can use a batch delete with condition on the orderlines table and use this where=(order_id,eq,101) and it will delete all the orderlines for order 101. After that you can delete order 101.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1054688140289376266)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by Charuru** [comment]
Is there a fix for this?
